en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline.md

```diff
@@ -315,6 +315,33 @@ docker build -t <IMAGE_NAME> -f <DOCKER_FILE_PATH> <CONTEXT>
 docker build -t myimages/choreo-connect-router:{{choreo_connect.version}} -f Dockerfile .
 ```
 
+### Mount Dropins Directory to a Choreo Connect Helm Deployment
+
+1. If you are using Choreo Connect as a Kubernetes Helm deployment, you can mount the dropins directory by following below instructions. 
+   
+      1. Create a configmap file to define the JAR that you are using with the Enforcer.
+
+        ```bash
+         kubectl create configmap configMapName1 --from-file=path/to/jar
+        ```
+
+      2. In the `values.yaml` file, update the `wso2.deployment.gatewayRuntime.enforcer.dropins` array with your configmap name(s).
+
+        ```bash tab='Example'
+         wso2:
+            deployment:
+                gatewayRuntime:
+                    enforcer:
+                        dropins:
+                            - configMapName1
+                            - configMapName2
+        ```
+
+2. If you are having a customised Enforcer Image (explained in the Production Deployment Guideline), follow the steps explained below considering the `values.yaml` file to mount the dropins folder. 
+
+      1. Define `wso2.deployment.gatewayRuntime.enforcer.dropinsMountEmptyDir` value as `false`.
```

Comment:
```suggestion
      1. Set `wso2.deployment.gatewayRuntime.enforcer.dropinsMountEmptyDir` value as `false`. Otherwise, the droppings already in the Docker image will be replaced with a Kubernetes empty dir.
```